t/143-ssl-session-fetch.t

```diff
@@ -1036,7 +1048,8 @@ qr/\S+:\d+: ssl fetch sess by lua is running!/s
         server_name   test.com;
         ssl_certificate $TEST_NGINX_CERT_DIR/cert/test.crt;
         ssl_certificate_key $TEST_NGINX_CERT_DIR/cert/test.key;
-        ssl_protocols SSLv3;
+        ssl_protocols TLSv1.2;
```

Comment:
Good point, since we use `ssl_session_tickets off` now there's no point in setting `ssl_protocols`. Removed now.